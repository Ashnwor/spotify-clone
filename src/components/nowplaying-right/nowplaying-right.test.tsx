import React from 'react';
import { shallow } from 'enzyme';
import NowPlayingRight from './nowplaying-right';

it('runs correctly', () => {
  const wrapper = shallow(<NowPlayingRight />);
  expect(wrapper).toMatchSnapshot();
});
