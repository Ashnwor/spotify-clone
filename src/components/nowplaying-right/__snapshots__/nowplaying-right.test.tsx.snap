// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly 1`] = `
<div
  className="nowplaying-right-container"
>
  <div
    className="nowplaying-right-playlist"
  >
    <SvgPlaylist
      className="nowplaying-right-icon"
    />
  </div>
  <div
    className="nowplaying-right-devices"
  >
    <SvgDevices
      className="nowplaying-right-icon-devices"
    />
  </div>
  <div
    className="nowplaying-right-volume"
  >
    <SvgVolume
      className="nowplaying-right-icon"
    />
  </div>
  <div
    className="nowplaying-right-volume-bar"
  />
</div>
`;
